Sources/NIOTLSServer/main.swift

```diff
@@ -29,6 +30,17 @@ private final class EchoHandler: ChannelInboundHandler {
     }
 }
 
+
+func pskCallback(hint: NIOSSLSecureBytes?, identity: NIOSSLSecureBytes?) -> (key: NIOSSLSecureBytes, identity: NIOSSLSecureBytes) {
+    var psk = NIOSSLSecureBytes()
+    var pskIdentity = NIOSSLSecureBytes()
+    psk.append(contentsOf: Data("hello".utf8))
```

Comment:
We don't need `Data` here, we can pass `hello.utf8` directly.

Reply:
Addressed in 308299a